src/components/terms-and-conditions.tsx

```diff
@@ -0,0 +1,205 @@
+// Terms and Conditions Pop-up
+
+import React, { ReactNode, useState } from "react";
```

Comment:
We dont need this import statement as Nextjs and React 18 automatically import this for us